rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -2099,22 +2099,39 @@ def test_object_download_at_replicated_site(
         else:
             zone_name = "primary"
         log.info(f"remote zone is {zone_name}")
+        if config.remote_zone == "archive":
+            zone_name = "archive"
+        log.info(
+            f"Download object {s3_object_name} via boto3 at remote site {zone_name}"
+        )
         # Download objects from remote site using boto3 rgw client
         remote_ip = utils.get_rgw_ip_zone(zone_name)
         remote_site_ssh_conn = utils.connect_remote(remote_ip)
         remote_site_auth = Auth(
             each_user, remote_site_ssh_conn, ssl=config.ssl, haproxy=config.haproxy
         )
         remote_s3_client = remote_site_auth.do_auth_using_client()
-
-        log.info(
-            f"Download object {s3_object_name} via boto3 at remote site {zone_name}"
+        if zone_name == "archive":
+            log.info(f"It is a {zone_name} zone, hence objects are always versioned.")
+        time.sleep(20)
```

Comment:
Any reason to induce delay here? we are already wait for sync in #L209

Reply:
@mkasturi18, we don't get the 'versions' in the response details of the object.